include/deal.II/dofs/dof_accessor.h

```diff
@@ -1260,10 +1260,10 @@ class DoFInvalidAccessor : public InvalidAccessor<structdim, dim, spacedim>
    * semantic sense, and we generate an exception when such an object is
    * actually generated.
    */
-  DoFInvalidAccessor(const Triangulation<dim, spacedim> *parent     = 0,
-                     const int                           level      = -1,
-                     const int                           index      = -1,
-                     const AccessorData *                local_data = 0);
+  DoFInvalidAccessor(const void *        parent     = 0,
+                     const int           level      = -1,
+                     const int           index      = -1,
+                     const AccessorData *local_data = 0);
```

Comment:
```suggestion
  DoFInvalidAccessor(const void *        parent     = nullptr,
                     const int           level      = -1,
                     const int           index      = -1,
                     const AccessorData *local_data = nullptr);
```
I'm not sure why clang-tidy didn't complain before - these should definitely be `nullptr`.